examples/format-dynamic-content.js

```diff
@@ -5,7 +5,7 @@ const winston = require('../');
 const logger = module.exports = winston.createLogger({
   transports: [new winston.transports.Console()],
   format: winston.format.combine(
-    winston.format(function dynamicContent(info, opts) {
+    winston.format(function dynamicContent(info) {
```

Comment:
I'm going to have to put my review on long pause here, because it looks like the PR has a lot of changes that aren't really part of the core proposed change, where figuring out the reasoning & potential implications (esp. on breaking things that work now) is going to be some significant amount of effort that I have no funding to work on.